client/components/Admin/App/SecuritySettings.tsx

```diff
@@ -19,7 +19,7 @@ const SecuritySettings: FC<Props> = ({ registrationMode: registrationModeOptions
   const [basicName, setBasicName] = useState(settingForm['security:basicName'] || '')
   const [basicSecret, setBasicSecret] = useState(settingForm['security:basicSecret'] || '')
   const [registrationMode, setRegistrationMode] = useState(settingForm['security:registrationMode'] || '')
-  const [registrationWhiteList, setRegistrationWhiteList] = useState(settingForm['security:registrationWhiteList'])
+  const [registrationWhiteList, setRegistrationWhiteList] = useState(settingForm['security:registrationWhiteList'].join(`\n`))
```

Comment:
The type of `security:registrationWhiteList` is `string[]`.

However, in the textarea, we used like `string`

https://github.com/crowi/crowi/blob/989dec5bf7a4555d78bf9a998e5c25e8faa54702/client/components/Admin/App/SecuritySettings.tsx#L80-L86